packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -463,19 +475,10 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
         return StoreInterface(FinderInterface(bridgeAdmin.finder()).getImplementationAddress(OracleInterfaces.Store));
     }
 
-    function _getProposerRewardAmount(uint64 _proposerRewardPct, uint256 _amount) private pure returns (uint256) {
+    function _getAmountFromPct(uint64 _RewardPct, uint256 _amount) private pure returns (uint256) {
```

Comment:
made a more general method for computing a % from a number.

Reply:
two nits:
- casing of `_RewardPct` should be `_rewardPct` I think.
- variable name implies this is a reward, but the function is meant to be general. Maybe it should be something more generic, like `_percent` or `_pct`?

Reply:
good points. will refactor accordingly.